firmware/src/model/weatherDataModel.cpp

```diff
@@ -53,7 +53,7 @@ float WeatherDataModel::getCurrentTemperature() {
 }
 
 String WeatherDataModel::getCurrentTemperature(int8_t digits) {
-    return Utils::formatFloat(m_currentWeatherDeg, digits);
+    return Utils::formatFloat(m_currentWeatherDeg, digits) + "°";
```

Comment:
I'd suggest to not hardcode this here, but either make it a member of the class or a define somewhere.

Reply:
Do you mean to define the Degree symbol somewhere else?
What's the point of that? If we want to change the temperature to Kelvin or something?